packages/json/src/index.ts

```diff
@@ -0,0 +1,38 @@
+import type { ZWaveLogInfo } from "@zwave-js/core";
+import * as NodeStream from "stream";
+import Transport from "winston-transport";
+
+export interface transportOptions {
+	level?: string;
+}
+
+export class JSONTransport extends Transport {
+	passThroughStream: NodeStream.PassThrough;
+	formattedMessageSymbol = Symbol.for("message");
+	public constructor(options: transportOptions) {
+		super({
+			level: options.level || "silly",
```

Comment:
AFAIK this MUST be `"silly"` or changes to zwave-js's loglevel won't propagate properly.